src/aws/cloudwatch-sink.cpp

```diff
@@ -415,6 +417,13 @@ void CloudWatchSink::init_context_info(nlohmann::json& dst,
     {
         dst["session_id"] = log.extra_identifier();
     }
+
+    if (log_thread_id_)
+    {
+        std::stringstream ss;
+        ss << std::this_thread::get_id();
+        dst["thread_id"] = ss.str();
```

Comment:
please use fmt format, faster

Reply:
not supported

Reply:
do we have a faster way?

Reply:
there is a hash function, but I don't think it will be faster
moved to ostringstream.
also, we already have much more complicated calculations so....

Reply:
i dont want to add to that...but w/e :D